Sources/Contentful/Decodable.swift

```diff
@@ -114,7 +127,7 @@ public extension KeyedDecodingContainer {
      */
     public func resolveLink(forKey key: KeyedDecodingContainer.Key,
                             decoder: Decoder,
-                            callback: @escaping (Any) -> Void) throws {
+                            callback: @escaping (AnyObject) -> Void) throws {
```

Comment:
Is the change from `Any` to `AnyObject` due to the Swift bug you reported yesterday?

Reply:
yup!

Reply:
there should be no affect on user code btw, but this is also for v4 which has other breaking changes so shouldn't matter